main.cpp

```diff
@@ -52,85 +52,94 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: car
+//  action 1: the car accelerates
+car.accelerate();
+//  action 2: the. car brakes
+car.brake();
+//  action 3: the car stops
+car.stop();
  
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: bird
+//  action 1: the bird flys
+bird.fly();
+//  action 2: the bird flaps wigs
+bird.flapWings();
+//  action 3: the bird lays an egg
+bird.layEgg();
  
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
+//  Noun: wind
+//  action 1: the wind increases
+wind.increase();
+//  action 2: the wind decreases
+wind.decrease();
+//  action 3: the wind stops
+wind.stop();
 
-//  action 3:
- 
 //  4)
-//  Noun:
-//  action 1:
+//  Noun: phone
+//  action 1: the phone rings
+phone.ring();
+//  action 2: the phone sends a notification
+phone.sendNotification();
+//  action 3: the phone restarts
+phone.restart();
 
-//  action 2:
-
-//  action 3:
- 
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
+//  Noun: iceCream
+//  action 1: the ice cream freezes
+iceCream.freeze();
+//  action 2: the ice cream melts
+iceCream.melt();
+//  action 3: the ice cream drips
+iceCream.drip();
 
-//  action 3:
- 
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: computer 
+//  action 1: the computer starts up
+computer.startUp();
+//  action 2: the computer computes
+computer.compute();
+//  action 3: the computer opens a file
+computer.openFile();
  
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
+//  Noun: light
+//  action 1: the light flashes
+light.flash();
+//  action 2: the light illuminates
+light.illuminate();
+//  action 3: the light EXPLODES!
+light.explode();
 
-//  action 3:
- 
 //  8)
+//  Noun: turtle
+//  action 1: the tutrle swims
+turtle.swim();
+//  action 2: the turtle walks
+turtle.walk();
+//  action 3: the turtle hides
+turtle.hide(); 
```

Comment:
As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
**REPLACE** the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.
**Do not** create an 11th noun.